src/App.js

```diff
@@ -43,10 +79,18 @@ class App extends React.Component {
 					<HeaderComp />
```

Comment:
No need to add "Comp" at the end of the component's name. It's obvious from its use.